manifests/m/Microsoft/VisualStudio/Locator/3.0.3.9285/Microsoft.VisualStudio.Locator.yaml

```
@@ -0,0 +1,7 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudio.Locator
+PackageVersion: 3.0.3.9285
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.1.0
```

```suggestion
ManifestVersion: 1.2.0
```
`1.2.0` schema includes support for `InstallerType: portable`